### DISCUSSION TITLE
2.16.0

### DISCUSSION THREAD

**Discussion by Karlatemp** [mamoe/mirai#2815] [Releases]
感谢大家四年以来对 mirai 的支持。阳光穿过秋天的枫叶，留下一张张独特的风景，如此灿烂，如此艳丽。

### 优化和修复

- 修复登录收到结果 180 时会无限重连的问题
- 修复 `ShortVideo` 文档
- 优化 `ShortVideo` 的内部日志

### 优化和修复

- 修复 JVM 插件 JAR 中使用 `plugin.yml` 加载时无法加载的问题 (#2791 by @cssxsh)

<hr /><em>This discussion was created from the release <a href='https://github.com/mamoe/mirai/releases/tag/v2.16.0'>2.16.0</a>.</em>

**Comment by kagg886** [comment]
一股要跑路的感觉(

**Reply by Longtianmu** [reply]
+1

**Reply by Takeoff0518** [reply]
QaQ

**Reply by qfys521** [reply]
感觉也差不多快润了()

**Comment by PieceOfFall** [comment]
mirai！没有你我怎么过啊，mirai！

**Comment by bobpanda** [comment]
不要跑路阿

**Comment by Xinezi** [comment]
论坛已经上不去了 你不要跑路啊 ！！！

**Reply by MrXiaoM** [reply]
论坛的域名因为没有实名被拒绝解析了，等几天him188搞好

**Comment by bingqiu456** [comment]
感谢开发者 感谢陪伴
**Mirai**

**Comment by AAAAQWERTY666** [comment]
燕子你别走

**Comment by AAAAQWERTY666** [comment]
燕子你别走
